### PR TITLE
State Versions API Docs

### PR DESCRIPTION
**Pull request by beekus**
screenshot-localhost-4567-2018 08 28-15-37-19

### CODE REVIEW

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Properties without a default value are required.
+
+
+Key path                    | Type    | Default   | Description
+----------------------------|---------|-----------|------------
+`data.type`                 | string  |           | Must be `"state-versions"`.
+`data.attributes.serial`    | integer |           | The serial of the state version
+`data.attributes.md5`       | string  |           | An MD5 hash of the raw state version
+`data.attributes.lineage`   | string  | (nothing) | Lineage of the state version
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/api/state-versions.html.md]: What does default of nothing mean here?

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
...CAgICAgICAgICAgICAgICAgImlk\nIjogIlFtX001Q283RTBBIiwKICAgICAgICAgICAgICAgICAgICAgICAgImF0\ndHJpYnV0ZXMiOiB7CiAgICAgICAgICAgICAgICAgICAgICAgICAgICAiYjY0\nIjogIlFtX001Q283RTBBIiwKICAgICAgICAgICAgICAgICAgICAgICAgICAg\nICJiNjRfc3RkIjogIlFtL001Q283RTBBPSIsCiAgICAgICAgICAgICAgICAg\nICAgICAgICAgICAiYjY0X3VybCI6ICJRbV9NNUNvN0UwQSIsCiAgICAgICAg\nICAgICAgICAgICAgICAgICAgICAiYnl0ZV9sZW5ndGgiOiAiOCIsCiAgICAg\nICAgICAgICAgICAgICAgICAgICAgICAiZGVjIjogIjQ3ODcyNzAyMDkyNTEy\nNTEwMDgiLAogICAgICAgICAgICAgICAgICAgICAgICAgICAgImhleCI6ICI0\nMjZmY2NlNDJhM2IxMzQwIiwKICAgICAgICAgICAgICAgICAgICAgICAgICAg\nICJpZCI6ICJRbV9NNUNvN0UwQSIsCiAgICAgICAgICAgICAgICAgICAgICAg\nICAgICAia2VlcGVycy4lIjogIjEiLAogICAgICAgICAgICAgICAgICAgICAg\nICAgICAgImtlZXBlcnMudXVpZCI6ICI0OGIwOGExOS02ODA1LTIxOTctYzMw\nYS04ZmM4ZjVmOWM2MTEiCiAgICAgICAgICAgICAgICAgICAgICAgIH0sCiAg\nICAgICAgICAgICAgICAgICAgICAgICJtZXRhIjoge30sCiAgICAgICAgICAg\nICAgICAgICAgICAgICJ0YWludGVkIjogZmFsc2UKICAgICAgICAgICAgICAg\nICAgICB9LAogICAgICAgICAgICAgICAgICAgICJkZXBvc2VkIjogW10sCiAg\nICAgICAgICAgICAgICAgICAgInByb3ZpZGVyIjogInByb3ZpZGVyLnJhbmRv\nbSIKICAgICAgICAgICAgICAgIH0KICAgICAgICAgICAgfSwKICAgICAgICAg\nICAgImRlcGVuZHNfb24iOiBbXQogICAgICAgIH0KICAgIF0KfQo=\n"
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/api/state-versions.html.md]: Wonder if we should put ... in here to shorten it.

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+```json
+{
+    "data": {
+        "id": "sv-SDboVZC8TCxXEneJ",
+        "type": "state-versions",
+        "attributes": {
+            "vcs-commit-sha": null,
+            "vcs-commit-url": null,
+            "created-at": "2018-08-27T14:49:47.902Z",
+            "hosted-state-download-url": "https://archivist.terraform.io/v1/object/........",
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/api/state-versions.html.md]: I think we conventionally put three dots for elisions, although when I checked another place we use archivist URLs, we just have the full ID.

**Comment by armchairlinguist** [comment]
@beekus Were we not going to include the "list state versions" endpoint?

File: content/source/docs/enterprise/api/state-versions.html.md (hunk 1 above)
Comment (beekus) [reply content/source/docs/enterprise/api/state-versions.html.md]: Just that it isn't required. Earlier versions of Terraform did not have lineage.

https://github.com/hashicorp/terraform-website/blob/master/content/source/docs/enterprise/api/_template.md#request-body

Comment (armchairlinguist) [reply content/source/docs/enterprise/api/state-versions.html.md]: I wonder if it's worth clarifying that, and maybe what purpose the lineage serves? I don't think we'd want people to create new states without lineage, right?

File: content/source/docs/enterprise/api/state-versions.html.md (hunk 2 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/api/state-versions.html.md]: 👍

File: content/source/docs/enterprise/api/state-versions.html.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+{
+    "data": [
+        {
+            "id": "sv-SDboVZC8TCxXEneJ",
+            "type": "state-versions",
+            "attributes": {
+                "vcs-commit-sha": null,
+                "vcs-commit-url": null,
+                "created-at": "2018-08-27T14:49:47.902Z",
+                "hosted-state-download-url": "https://archivist.terraform.io/v1/object/..."
```
Comment (nfagerlund) [inline content/source/docs/enterprise/api/state-versions.html.md]: Missing a comma after this item.

**Pull request merged**